Microsoft.Toolkit.Uwp.UI.Controls/GridSplitter/GridSplitter.Events.cs

```diff
@@ -22,30 +24,22 @@ namespace Microsoft.Toolkit.Uwp.UI.Controls
     /// </summary>
     public partial class GridSplitter
     {
+        // Symbols for GripperBar in Segoe MDL2 Assets
+        private const string GripperBarVertical = "\xE784";
```

Comment:
While I see some room for improvement here, I do not agree that the GridSplitter should be a ContentControl. A ContentControl is a control that is primarily used to display content. This is absolutely not the case with GridSplitter. The "Element" property is only there to allow easy re-styling of the gripper. I think the naming is rather bad, it should be something like GripperIcon or whatever. But making the control a ContentControl? No, I'd rather not.

Besides that, I think this is a bit beyond the scope of this PR. It is not something that can be easily be done along with the fix, this would be more like a major redesign of the whole control. I have only moved this piece of code to a different place (it was in GridSplitterGripper.cs which is now deleted). If someone wants to redesign the control, can you do this in a separate PR? I do not really see the need for this, and to be honest, I also do not feel very comfortable yet working with visual states. It would be great if someone else could do this, if you think that this area really must be redesigned.